miqcli/constants.py

```diff
@@ -63,9 +63,47 @@
     'enable_ssl_verify': False
 }
 
+OS_TYPE = "ManageIQ::Providers::Openstack::CloudManager"
+OS_NETWORK_TYPE = "ManageIQ::Providers::Openstack::NetworkManager"
+
+
+SUPPORTED_AUTOMATE_TYPES = ["gen_floating_ip"]
+
+SUPPORTED_PROVIDERS = ["OpenStack"]
+REQUIRED_OS_KEYS = ["email", "tenant", "image", "security_group", "network",
+                    "flavor", "key_pair", "vm_name"]
+OPTIONAL_OS_KEYS = ["fip_pool"]
+
 #: filesystem root for miqcli authentication store
 AUTHDIR = os.path.join(os.path.expanduser('~'), ".miqcli/auth")
 
+#: time to wait in secs in between checks of a task
```

Comment:
Can you explain more what task is and why 2 secs is the best for default? What happen if I add 10 secs?